src/main/java/com/dougnoel/sentinel/elements/Element.java

```diff
@@ -712,14 +712,31 @@ public boolean attributeEquals(String attribute, String value) {
 			String values = element().getAttribute(attribute);
 			log.debug("Values found for attribute {} on element {}: {}", attribute, this.getClass().getName(),
 					values);
-			if (values.equals(value)) {
+			return values.equals(value);
+		}
+		return false;
+	}
+
+	/**
+	 * Returns true if the element has an attribute containing to the value passed;
+	 * otherwise returns false.
+	 * <p>
+	 * <b>Examples:</b>
+	 * <ul>
+	 * <li>Determine if "blob-code-context" is within an element that has a class="blob-code blob-code-context js-file-line" attribute set.
+	 * </ul>
+	 *
+	 * @param attribute String the attribute to look for
+	 * @param value String the value to which attribute should be set
+	 * @return boolean true if the element as an attribute containing the value passed; otherwise returns false
+	 */
+	public boolean attributeContains(String attribute, String value) {
+		String values = element().getAttribute(attribute);
+		log.debug("Values found for attribute {} on element {}: {}", attribute, this.getClass().getName(),
+				values);
+		for (String c : values.split(" ")) {
+			if (c.equals(value)) {
 				return true;
-			} else {
```

Comment:
I did eventually break this out into its own function because how "has" should mean equals, when before it was a soft contains instead. Adding the negative case here would have had issues.